frameworks/Go/go-raw/src/hello/hello.go

```diff
@@ -48,11 +48,17 @@ const (
 	connectionString   = "benchmarkdbuser:benchmarkdbpass@tcp(localhost:3306)/hello_world?interpolateParams=true"
```

Comment:
Does `interpolateParams=true` affect prepared statement version? I assume that no